docs/quickstart/avalanche-faucet.md

```diff
@@ -0,0 +1,428 @@
+
+* **Timeout of 10000ms exceeded**
+There could be many reasons for this message. It could be an internal server error, or the request didn't receive by the server, slow internet, etc. You could try again after some time, and if the problem persists, then you should raise this issue on our [Discord](https://discord.com/channels/578992315641626624/) server.
+
+* **Couldn't see any transaction status on explorer**
+The transaction hash that you get for each drop is pre-computed using the expected nonce, amount, and receiver's address. Though transactions on Avalanche are near-instant, the explorer may take time to index those transactions. You should wait for a few more seconds, before raising any issue or reaching out to us.
+
+## Adding a New Subnet
+
+You can also integrate a new Subnet on the live [faucet](https://faucet.avax.network) with just 15 lines of configuration parameters. All you have to do is make a PR on the [Avalanche Faucet](https://github.com/ava-labs/avalanche-faucet) git repository with the Subnet's information. The following parameters are required.
+
+```json
+{
+    "ID": string,
+    "NAME": string,
+    "TOKEN": string,
+    "RPC": string,
+    "CHAINID": number,
+    "EXPLORER": string,
+    "IMAGE": string,
+    "MAX_PRIORITY_FEE": string,
+    "MAX_FEE": string,
+    "DRIP_AMOUNT": number,
+    "RATELIMIT": {
+        "MAX_LIMIT": number,
+        "WINDOW_SIZE": number
+    }
+}
+```
+
+* **ID** - Each subnet chain should have a unique and relatable ID.
+* **NAME** - Name of the Subnet chain that will appear on the website.
+* **RPC** - A valid RPC URL for accessing the chain.
+* **CHAINID** - Chain ID of the chain
+* **EXPLORER** - Base URL of standard explorer's website.
+* **IMAGE** - URL of the icon of the chain that will be shown in the dropdown.
+* **MAX_PRIORITY_FEE** - Maximum tip per faucet drop in **wei** or **10<sup>-18</sup>** unit (for EIP1559 supported chains)
+* **MAX_FEE** - Maximum fee that can be paid for a faucet drop in **wei** or **10<sup>-18</sup>** unit
+* **DRIP_AMOUNT** - Amount of coins to send per request in **gwei** or **10<sup>-9</sup>** unit
+* **RECALIBRATE** *(optional)* - Number of seconds after which the nonce and balance will recalibrate
+* **RATELIMIT** - Number of times (MAX_LIMIT) to allow per user within the WINDOW_SIZE (in minutes)
+
+Add the configuration in the array of `evmchains` inside the [config.json](https://github.com/ava-labs/avalanche-faucet/blob/main/config.json) file and make a PR.
+
+## Building and Deploying Faucet
+
+You can also deploy and build your faucet by using the [Avalanche Faucet](https://github.com/ava-labs/avalanche-faucet) repository.
+
+## Requirements
```

Comment:
Should this section and following sections be under `Building and Deploying Faucet` which means it should be 
```suggestion
### Requirements
```